preferences-commons/src/main/java/no/sikt/nva/person/preferences/commons/model/PersonPreferences.java

```diff
@@ -13,7 +14,8 @@
 @JsonSerialize
 public record PersonPreferences(URI personId,
                                 List<URI> promotedPublications,
-                                @JsonIgnore PersonPreferencesService personPreferencesService) {
+                                @JsonIgnore PersonPreferencesService personPreferencesService,
```

Comment:
We do not want to add created and modified dates to PersonPreferences, but only Dao

Reply:
Removed.